tools/gromacs/macros.xml

```diff
@@ -3,9 +3,9 @@
     <xml name="requirements">
         <requirements>
             <requirement type="package" version="@TOOL_VERSION@">gromacs</requirement>
+        <yield/>   
```

Comment:
```suggestion
```
Can you remove this if it's not needed for this PR?

Reply:
Ok did that.